src/main/resources/labels/labels.json

```diff
@@ -5671,6 +5671,96 @@
         "multilangLabel": { }
     },
     {
+        "uri": "http://purl.org/lobid/lv#contributorOrder",
+        "label": "contributorOrder",
```

Comment:
This is already part of the context-labels, see https://github.com/hbz/lobid-resources/blob/72-MARCRelatorCodesInMABRDARepair/src/main/resources/labels/context-labels.json